plugins/s3_auth/s3_auth.cc

```diff
@@ -711,6 +746,7 @@ TSRemapDoRemap(void *ih, TSHttpTxn txnp, TSRemapRequestInfo * /* rri */)
 
   if (s3) {
     TSAssert(s3->valid());
+    s3->acquire(); // Increasement ref-count
```

Comment:
There is a race condition here, but it is >60s and something we take as a known risk in the code as it is